ion-c-sys/src/lib.rs

```diff
@@ -596,9 +599,12 @@ mod test_bigdecimal {
                 assert_eq!(exponent, actual_exponent, "Testing exponents");
             }
             Err(e) => match e.code {
-                ion_error_code_IERR_INVALID_STATE => {
-                    println!("Ignoring amzn/ion-rust#79 for {}", d_lit)
-                }
+                ion_error_code_IERR_INVALID_STATE => match e.additional {
+                    "DecNumber Not Supported" => {
```

Comment:
Since we're producing and later matching on this `&str` literal, I think we should make it a constant.

Reply:
Done.